Mobile/app/src/main/java/com/example/mobile/api/ApiCall.kt

```diff
@@ -107,7 +109,7 @@ class ApiCall {
                 val json = Gson().toJson(dto)
                 postApiCall(json, url) { result, e ->
                     if (e != null) {
-                        callback(null, Exception("Invalid username or password"))
```

Comment:
aici eu as lasa Invalid username or password, pentru ca eroarea care vine de acolo e ciudata si e ciudat sa o alertam. dar eventual putem lasa asa momentan si vedem schimbam pe parcurs

Reply:
nu prea e corect sa lasam asta, pt ca imi crapa functia asta si venea o eroare cu totul diferita.
mai rau m-a indus in eroare chestia asta:))
eu zic ca mai bine afisam eroarea care vine, pt ca e gresit sa presupunem ca este invalid username sau pass